app/models/miq_policy.rb

```diff
@@ -11,12 +11,26 @@ class MiqPolicy < ApplicationRecord
                                  PhysicalServer
                                  Vm).freeze
 
-  POLICY_TOWHAT_APPLIES_TO_CLASSES = TOWHAT_APPLIES_TO_CLASSES.inject({}) { |hsh, key| hsh[key] = ui_lookup(:model => key); hsh }.freeze
-
-  POLICY_MODES = {
-    "control" => _("Control"),
-    "compliance" => _("Compliance")
-  }.freeze
+  def self.POLICY_TOWHAT_APPLIES_TO_CLASSES(opts)
```

Comment:
While it's tempting to keep this for cross-repo compatilibtiy, this should become a underscored method name
```suggestion
  def self.policy_towhat_applies_to_classes(opts)
```